apps/fis/perftest/base/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../../rbac/nonprod-role.yaml
+namespace: fis
```

Comment:
Make changes to automation folder too to automate the process. Else it won't reflect